src/ffi.rs

```diff
@@ -151,7 +151,7 @@ const I2C_RDRW_IOCTL_MAX_MSGS: u8 = 42;
 
 /// This is the structure as used in the I2C_SMBUS ioctl call
 #[repr(C)]
-struct i2c_smbus_ioctl_data {
+pub struct i2c_smbus_ioctl_data {
```

Comment:
Up to you - this could potentially go in the `mod ioctl` below

Reply:
I started to play around with this and it ended up requiring a bunch more changes.  I think I'm OK with this leaking since `ffi` isn't pub itself and is just used internally.